View.java

```diff
@@ -0,0 +1,159 @@
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Font;
+import java.awt.FontFormatException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.awt.event.ActionListener;
+import java.awt.event.InputMethodListener;
+import java.beans.PropertyChangeListener;
+import java.awt.event.ActionEvent;
+
+import javax.swing.*;
+import javax.swing.event.ChangeListener;
+import javax.swing.event.RowSorterListener;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import javax.swing.table.DefaultTableModel;
+
+
+
+public class View extends JFrame {
+	
+	Dimension size = new Dimension(1050,700);
+	JTextField text = new JTextField();
+	Font font;
+	Font font2;
+	File filefont;
+	String[] list = { "Title", "Genre", "Director", "Publication Date", "Length" };
+	ArrayList<String> dataa = new ArrayList<String>();
+	
+	private JPanel mainPanel = new JPanel();
+	private JPanel navPan = new JPanel();
+
+	private JButton addfromfile = new JButton("Add from CSV file");
+	private JButton savebutton= new JButton("Save to");
+	private JButton LoadFile = new JButton("Load CSV File");
+	private JButton add = new JButton("Add");
+	private JButton delete = new JButton("Delete");
+	private JComboBox sortList = new JComboBox(list);
+	private JButton sortup = new JButton ("⇅");
+	private JLabel search = new JLabel("Search");
+	private JTextField searchfield = new JTextField();
+	private JLabel searchby = new JLabel ("Search by");
+	private JPanel bottompan = new JPanel();
+	private DefaultTableModel tabmodel= new DefaultTableModel(list, 0);
+	private JTable table = new JTable(tabmodel);
+	private JScrollPane scroll = new JScrollPane(table);
+	private JFileChooser chooseFile = new JFileChooser();
+	private FileNameExtensionFilter csvfilter = new FileNameExtensionFilter("CSV File", "csv");

```

Comment:
Czy te wszystkie pola są na pewno potrzebne? Bo jeśli nie są przekazywane nigdzie to można je zadeklarować jako lokalne zmienne